frontend_vue/src/components/base/BaseCodeSnippet.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div>
+    <label
+      v-if="label"
+      class="inline-block mb-8 text-grey-500"
+      :for="label"
+      >{{ label }}</label
+    >
+
+    <div class="relative border rounded-lg border-grey-100">
+      <div class="absolute top-[.8rem] right-[1rem] z-10 flex gap-8">
+        <BaseRefreshButton
+          v-if="hasRefresh"
+          @refresh-token="handleRefreshToken"
+        />
+        <BaseCopyButton :content="code" />
+      </div>
+      <VCodeBlock
+        :id="label"
+        :code="code"
+        highlightjs
+        :lang="lang"
+        theme="a11y-light"
```

Comment:
do we like this theme? it feels a little christmassy. what about something more githubesque?

Reply:
christmassy 😂 no yeah for js it's true, it also changes depending on the language
( this for instance is the snippet for php 
<img width="999" alt="Screenshot 2024-03-27 at 15 04 23" src="https://github.com/thinkst/canarytokens/assets/126554007/df5945be-cefe-430b-ba74-5511d1de7ec5">

But we can use another one, I'm not opinionated about it
https://webdevnerdstuff.github.io/vue-code-block/#props

I would just keep one with a light background

Reply:
can we try the github theme?

Reply:
I think it looks fine 

JS:
<img width="997" alt="Screenshot 2024-03-27 at 15 19 50" src="https://github.com/thinkst/canarytokens/assets/126554007/ce13061e-f06a-41f5-9d64-0f3cb25da159">